File: lat1.py
#PESAN
print('Berikut ini adalah aplikasi Matematika')
print('Berikut Pilihan Operasi:')

print("1. Operasi Tambah ( + )")
print("2. Operasi Kurang ( - )")
print("3. Operasi Kali ( * )")
print("4. Operasi Bagi ( / )")
print("5. Operasi Pangkat ( ^ )")
print('6. Hitung fahrenheit')
print('7. Hitung Luas Persegi')

def proses(operasi):
	if 1 <= operasi <=4:
	#INPUT
		angka_1 = int(input('Masukkan angka pertama : '))
		angka_2 = int(input('Masukkan angka kedua : '))
		if operasi == 1:
			print(angka_1 + angka_2)
		elif operasi == 2:
			print(angka_1 - angka_2)
		elif operasi == 3:
			print(angka_1 * angka_2)
		elif operasi == 4:
			print(int(angka_1 / angka_2))
	elif operasi == 5:
		angka_3 = float(input('Masukkan angka yang ingin di pangkatkan! : '))
		angka_4 = float(input('Masukkan angka pangkatnya! : '))
		print(f"Hasilnya : {angka_3 ** angka_4}")
	elif operasi == 6:
		celcius = float(input('Masukkan Celcius : '))
		print(f"Suhu Dalam fahrenheit : {(celcius *9/5)+32}")
	elif operasi == 7:
		lebar = int(input('Masukkan lebar! : '))
		panjang = int(input('Masukkan panjang! : '))
		print(f"Luas perseginya adalah: {lebar*panjang}")
	else:
		print('Pilihan Tersebut Tidak Ada')

try:
	operasi = int(input('Pilih Operasi Nomor Berapa ? : '))
	proses(operasi)
except ValueError:
	print('Terjadi Error Dengan pilihan yang di pilih')

File: lat1.js
const readline = require('readline')
const { stdin: input, stdout: output } = require('node:process');

const rl = readline.createInterface({
	input: process.stdin,
	output: process.stdout
})

rl.question(`1. Tambah ( + )
2. Kurang ( - )
3. Kali ( x )
4. Bagi ( : )
5. Hitung Luas Persegi
6. Hitung Fahrenheit

Silahkan pilih Operasi yang akan di gunakan : `, (data) => {
	if( data >= 1 && data <= 4) {
		console.log('Penjumlahan')
		 operasiPenjumlahan(data)
	}else if ( data == 5 ) {
		rl.question('Masukkan Lebar : ', (l) => {
			rl.question('Masukkan Tinggi : ', (t) => {
				console.log(`Luasnya adalah : ${l*t}`)
				rl.close()
			})
		})
	}else if( data == 6) {
		rl.question('Masukkan Celcius : ', (c) => {
			console.log(`Fahrenheit ${(c *9/5)+32}`)
			rl.close()
		})
	}else {
		rl.close()
	}
	
})



const operasiPenjumlahan = (data => {
	rl.question('Masukkan angka pertama : ', (c1) => {
		rl.question('Masukkan angka kedua : ', (c2) => {
			if( data == 1 || data == '+' ) {
				console.log(`Hasil Operasinya : ${c1+c2}`)
			}else if( data == 2 || data == '-' ) {
				console.log(`Hasil Operasinya : ${c1-c2}`)
			}else if (data == 3 || data == 'x') {
				console.log(`Hasil Operasinya : ${c1*c2}`)
			}else if (data == 4 || data == ':') {
				console.log(`Hasil Operasinya : ${c1/c2}`)
			}else{console.log('Maaf ada yang salah')}
			rl.close()
		})
	})
})